source/MaterialXRender/Handlers/TinyEXRImageLoader.h

```diff
@@ -0,0 +1,46 @@
+#ifndef MATERIALX_TinyEXRImageLoader_H
+#define MATERIALX_TinyEXRImageLoader_H
```

Comment:
Nitpick: We normally use all large caps for header guards.

Reply:
Search and replace error. I'll fix up.